partner_firstname/__openerp__.py

```diff
@@ -15,35 +17,34 @@
 #
 #    You should have received a copy of the GNU Affero General Public License
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
 
 {
-    'name': 'Partner first name, last name',
+    'name': 'Partner first name and last name',
```

Comment:
Would be great to add a `summary` key here.

Your title in description would fix perfectly
"Split first name and last name for non company partners"